src/generics.rst

```diff
@@ -173,9 +173,39 @@ A :t:`constant parameter` shall be used in the following contexts:
   As a :t:`value` of any :t:`expression` within a :t:`function body`.
 
 :dp:`fls_hidfwkwr2r73`
-A :t:`type parameter` has an implicit :std:`core::marker::Sized` bound, unless a
+A :t:`type parameter` has an implicit :std:`core::marker::Sized` :t:`bound`, unless a
 ``?core::marker::Sized`` bound is present.
 
+:dp:`fls_yaYedRNaxhTe`
+A :t:`type parameter` declared on an :t:`abstract data type` has implicit
+:t:`lifetime` :t:`[bound]s` depending on its usages in the :t:`abstract data
+type`'s fields as follows:
+
+* :dp:`fls_axMVssU7an8p`
+  If the :t:`type parameter` is used in a :t:`reference type` it has an implicit
```

Comment:
How about

If the type parameter is used in a reference type, then the type parameter has an implicit lifetime bound for the lifetime of the reference type.